pkg/model/defaults/volumes_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package defaults
+
+import (
+	"testing"
+
+	"k8s.io/kops/pkg/apis/kops"
+)
+
+func TestDefaultInstanceGroupVolumeSize(t *testing.T) {
+	tests := []struct {
+		role     kops.InstanceGroupRole
+		expected int32
+	}{
+		{
+			role:     "Master",
+			expected: 64,
+		},
+		{
+			role:     "Node",
+			expected: 128,
+		},
+		{
+			role:     "Bastion",
+			expected: 32,
+		},
+		{
+			role:     "Node2",
+			expected: -1,
```

Comment:
@geojaz this particular one could perhaps have some value. I suppose we want to preserve `-1` being the return value for unknown roles. The above could perhaps just test for positive integers instead.

Reply:
If we were to add another role, it would at least be nice to see this test failing.

Reply:
Great point +1 @olemarkus